### DISCUSSION TITLE
Ran the free version = Good. Bought the premium version = Bad.

### DISCUSSION THREAD

**Discussion by Leterel** [aclap-dev/video-downloadhelper#961] [License & Purchase requests]
### What's your issue?

Hello,

I downloaded videos from a NSFW site in the free version.
Worked perfectly.
Bought the premium version.
Everything stopped working.

Option 1: I get my money back.
Option 2: Help me to get this run.

### The 10 first digits (only 10) of your license number or email address

missgeburtensalato@web.de

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.  So I can't do anything about refunding your license.

But if you want to open a proper problem report, I might be able to help you.  You have to go to the `Help and Q&A` section of this forum & start a problem report.  Make sure you actually READ & FOLLOW the instructions there.  Post a complete description of your operating environment following the instruction provided during the process of creating your problem report.  Also, provide at least one URL of a web page on which you are having trouble.  Don't worry about NSFW.  I'm old enough that I won't be scandalized.  My mommy & daddy won't be revoking my computer privileges because of it.

**Reply by Leterel** [reply]
Fanks. :)

**Reply by jcfrog** [reply]
And don't worry, you have one month for the refund if needed.
